pkg/utils/pod.go

```diff
@@ -94,6 +94,11 @@ func IsStaticPod(pod *v1.Pod) bool {
 	return err == nil && source != "api"
 }
 
+// IsPriorityPod returns true if the pod is a priority pod.
+func IsPriorityPod(pod *v1.Pod) bool {
```

Comment:
Maybe `IsCriticalPriorityPod` just to be clear what kind of priority we're checking

Reply:
Good call here, I'll edit this.